proto/cosmos/feegrant/v1beta1/query.proto

```diff
@@ -53,3 +59,20 @@ message QueryAllowancesResponse {
   // pagination defines an pagination for the response.
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
+
+// QueryAllowancesByGranterRequest is the request type for the Query/AllowancesByGranter RPC method.
+message QueryAllowancesByGranterRequest {
+  string granter = 1;
```

Comment:
This differs from https://github.com/cosmos/cosmos-sdk/pull/10947/files#diff-b63bf05331253990f5ec6cd4279d8a2364bea0baa6c08778a88cc9290dbaddecR66 because #9933 is not implemented in this version.